background/background.js

```diff
@@ -511,25 +480,25 @@ if (typeof chrome.commands !== "undefined") {
             chrome.tabs.query({
                 currentWindow: true,
                 active: true
-            }, tabs => {
+            }, tabs =>
                 chrome.tabs.sendMessage(tabs[0].id, {
                     action: "TranslateSelectedText"
-                }, checkedLastError)
-            })
+                }, checkedLastError))
         } else if (command === "hotkey-swap-page-translation-service") {
             chrome.tabs.query({
                 active: true,
                 currentWindow: true
-            }, tabs => {
+            }, tabs =>
                 chrome.tabs.sendMessage(tabs[0].id, {
                     action: "swapTranslationService"
-                }, checkedLastError)
-            })
+                }, checkedLastError))
 
             let currentPageTranslatorService = twpConfig.get("pageTranslatorService")
             if (currentPageTranslatorService === "google") {
                 currentPageTranslatorService = "yandex"
-            } else {
+            } else if (currentPageTranslatorService === "yandex") { //I'm not sure in this part
+                currentPageTranslatorService = "bing"
+            } else if (currentPageTranslatorService === "bing") {
```

Comment:
This extension has only 2 page translation services. So this doesn't work.

Reply:
Then you can make it
`if (currentPageTranslatorService === "google") {
currentPageTranslatorService = "yandex"
} else {
currentPageTranslatorService = "google"
}`